README.md

```diff
@@ -8,13 +8,202 @@
+- It has a striped pattern.
+- It's been declawed.
+- It weighs 10lbs.
+
+You can think of a JavaScript Object like the "mental image" of this cat. A
+set of related properties that defines a single item.
+
+If this cat were a JavaScript Object, what would it look like?
+
+## Creating an Object
+
+We start simply, using the shorthand Object constructor: `{}`
+
+```javascript
+var cat = {};
+```
+
+Now we have a cat. What can we do with it?
+
+## Access a value
+
+Let's see what color our cat is. We'll do this by accessing variables available
+on the `cat` Object.
+
+Let's try:
+
+```
+cat.color;
+```
+
+And see what we get!
+
+Uh oh. `undefined`. That's not helpful. Of course it isn't because we forgot
+to actually tell our program what the value of the `color` property (just like
+  a variable) of `cat` actually is! How do we fix that?
+
+Easy. We can set any property we want on any object (in this case, `cat`) by
+simply writing...
+
+```
+cat.color = 'orange';
+```
+
+Great, and now we access `cat.color` and get... you guessed it, `'orange'`!
+
+Like I mentioned, this `color` is now an Object *property* and it corresponds to
+the value `'orange'`. We also refer to Object properties as *keys*, the combination of the
+key and value colloquially referred to as a *key-value pair*.
+
+Let's add the rest of our data. Note that we can use any variable type when
+assigning keys / properties to Objects.
+
+```javascript
+var cat = {};
+cat.color = 'orange'; // string
+cat.pattern = 'striped';
+cat.declawed = true; // boolean
+cat.weight = 10; // number
+```
+
+We can also use the shorthand Object constructor to write the equivalent:
+
+```javascript
+var cat = {
+  color: 'orange',
+  pattern: 'striped',
+  declawed: true,
+  weight: 10
+};
+```
+
+Or, alternatively, just use quotation marks:
+
+```javascript
+var cat = {};
+cat['color'] = 'orange'; // string
+cat['pattern'] = 'striped';
+cat['declawed'] = true; // boolean
+cat['weight'] = 10; // number
+```
+
+These methods are all identical, and the keys as strings example (above) is
+for when you have oddly named properties.
+
+## Modifying a Key-Value pair
+
+Once you've created your object, the properties (or keys) attached to your
+object are *mutable*, which means to say they can change. (Think of the word
+  mutation. You're mutating the value.) They can also be removed completely.
+
+If we take our example above, and we do:
+
+```
+console.log(cat.color);
+```
+
+We'll see the output
+
+```
+'orange'
+```
+
+We can change this with
+
+```
+cat.color = 'brown';
+```
+
+We can also remove the property altogether with:
+
+```
+delete cat.color;
+```
+
+Now,
+
+```
+console.log(cat.color) // => undefined
+```
+
+And if we log the cat Object, we get:
+
+```
+{
+  pattern: 'striped',
+  declawed: true,
+  weight: 10
+}
+```
+
+No color property to be found!
+
+## Iterating over Key-Value pairs
+
+To loop over every key-value pair in an Object, you can use a `for in` loop.
+
+```
+for (key in cat) {
```

Comment:
Students might not yet grasp basic `for` loops with arrays completely (they'll have learned the concept pretty recently), so it's really important to take this slowly.